src/main/java/org/example/repository/AccountRepositoryImpl.java

```diff
@@ -0,0 +1,108 @@
+package org.example.repository;
+
+import org.example.repository.dao.AccountRepository;
+
+import java.sql.*;
+
+public class AccountRepositoryImpl implements AccountRepository {
+
+    private Connection connection;
+    private final String add = "INSERT INTO accounts (type, amount) VALUES (?, ?)";
+    private final String get = "SELECT amount FROM accounts WHERE id = ?";
+    private final String increase = """
+            UPDATE accounts 
+            SET amount = amount + ?
+            WHERE id = ?
+            """;
+    private final String decrease = """
+            UPDATE accounts 
+            SET amount = amount - ?
+            WHERE id = ?
+            """;
+
+    public AccountRepositoryImpl(Connection connection) {
+        this.connection = connection;
+    }
+
+    @Override
+    public int create(String type, double amount) {
+        int generatedId = -1;
+
+        try (PreparedStatement statement = this.connection.prepareStatement(add, Statement.RETURN_GENERATED_KEYS)) {
+            statement.setString(1, type);
+            statement.setDouble(2, amount);
+            statement.execute();
+
+            ResultSet rs = statement.getGeneratedKeys();
```

Comment:
bad variable name. 
Such a short name is allowed to be used in Lambda Expressions and Exceptions

Reply:
Variable "rs" was changed to "resultSet"